### ISSUE TITLE
Hyperlink: Review `Accessibility` page/tab

### ISSUE THREAD

**Issue by jordanjones243** [AlaskaAirlines/auro#381]
# General Support Request

Need to review `Accessibility` page for content accuracy. Consult @blackfalcon during this process.
